docs/data/material/getting-started/mcp/mcp.md

```diff
@@ -28,14 +28,12 @@ The sections below detail how to set up the Material UI MCP in popular agentic
 Open the MCP configuration (**Settings** -> **MCP** -> **Add Server**) and add the following:
 
 ```json
-"mcp": {
-  "servers": {
+"mcpServers": {
     "mui-mcp": {
```

Comment:
The indentation seems off (4 instead of 2). I'm wondering why eslint doesn't kick in.